### PR TITLE
chore: simplify ficha component

### PR DESCRIPTION
**Pull request by juanigaray**
- We were making an API call when it wasn't really needed. Importing the corresponding JSONs shortens the code and prevents errors when building the URL.
- Several operations were made inside the Ficha component. This commit makes these operations named and declarative so the component is more easily read.
- We weren't handling a few cases in userland. The page showed a loading text when it failed and didn't report errors at all. Early returns provide a skeletal but clear error message.

### CODE REVIEW

**Review by alexaiconejo — COMMENTED** [review]
ok

File: src/components/fichas/Ficha.jsx [hunk 1]
```diff
@@ -1,65 +1,51 @@
...
         <div className={styles.data}>
           <h2 className={styles.title}>
-            {tipoCaso === "gatillo"
-              ? "Caso de gatillo fácil"
-              : tipoCaso === "dependencias"
-                ? "dependencia policial"
-                : tipoCaso === "reportes"
-                  ? "Reporte de violencia policial"
-                  : "Tipo Desconocido"}
+            {titleByTipoCaso(tipoCaso) ?? "Tipo Desconocido"}
```
Comment (serugeneris) [inline src/components/fichas/Ficha.jsx:48]: Es un objeto, no una función!
TypeError: titleByTipoCaso is not a function

**Review by serugeneris — CHANGES_REQUESTED** [review]

File: src/components/fichas/Ficha.jsx (hunk 1 above)
Comment (juanigaray) [reply src/components/fichas/Ficha.jsx:48]: sí, me saltó con TS. De hecho voy a mergear las dos ramas porque es un incordio si no

Comment (serugeneris) [reply src/components/fichas/Ficha.jsx]: oka, cierro entonce

**Review by serugeneris — APPROVED** [review]
A darle átomos!

**Pull request merged**